### PR TITLE
288 add info toast 1

### PR DESCRIPTION
**Pull request by Shojiki-Lukhozi**
# Description

Added Toasts To The Application

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/app/auth/data-access/src/auth.state.ts [hunk 1]
```diff
@@ -218,8 +223,10 @@ export class AuthState {
       cognito.changePassword(params, (err, data) => {
         if (err) {
           console.error('Password change error:', err);
+          this.store.dispatch(new ShowInfo("An Error Occurred When Chanaging Password"));
```
Comment (ryanbasiltrickett) [inline libs/app/auth/data-access/src/auth.state.ts:226]: Show error rather

File: libs/app/auth/data-access/src/auth.state.ts [hunk 2]
```diff
@@ -174,6 +177,7 @@ export class AuthState {
         this.store.dispatch(new RetrieveProfile(username));
         this.store.dispatch(new RetrievePreferences(username));
         this.store.dispatch(new GetUpdatedRecommendation(username));
+        this.store.dispatch(new ShowSuccess("Logged In Successfully"));
```
Comment (ryanbasiltrickett) [inline libs/app/auth/data-access/src/auth.state.ts:180]: Don't do this a user might not be successfully logged in if the API doesn't work correctly

**Review by ryanbasiltrickett — CHANGES_REQUESTED** [review]

File: libs/app/recipe/data-access/src/recipe.state.ts [hunk 3]
```diff
@@ -79,7 +81,7 @@ export class RecipeState {
         }
       },
       (error: Error) => {
-        console.error('Failed to retrieve recipe:', error);
+        console.error('Failed to retrieve recipe: ', error);
```
[the remark this answers is not in the document]
Comment (Shojiki-Lukhozi) [reply libs/app/recipe/data-access/src/recipe.state.ts:84]: Done

File: libs/app/recipe/data-access/src/recipe.state.ts [hunk 4]
```diff
@@ -70,6 +71,7 @@ export class RecipeState {
           patchState({
             recipe: recipe,
           });
+          this.store.dispatch(new ShowSuccess('Found Recipe'));
```
[the remark this answers is not in the document]
Comment (Shojiki-Lukhozi) [reply libs/app/recipe/data-access/src/recipe.state.ts:74]: Done

File: libs/app/profile/ui/src/password-modal/password-modal.component.ts [hunk 5]
```diff
@@ -28,7 +29,7 @@ export class PasswordModalComponent {
   save() {
 
     if(this.newPassword != this.confirmPassword)
-      this.store.dispatch( new ShowError("Please enter matching passwords"));
+      this.store.dispatch( new ShowInfo("Please enter matching passwords"));
```
[the remark this answers is not in the document]
Comment (Shojiki-Lukhozi) [reply libs/app/profile/ui/src/password-modal/password-modal.component.ts:32]: Done

File: libs/app/profile/data-access/src/profile.state.ts [hunk 6]
```diff
@@ -334,6 +336,8 @@ export class ProfileState {
             this.mealPlanAPI.saveMealPlan(mealPlan);
             this.store.dispatch( new RetrieveMealPlanIngredients(mealPlan) );
         }
+
+        this.store.dispatch(new ShowInfo('Meal Plan Has Been Updated'));
```
[the remark this answers is not in the document]
Comment (Shojiki-Lukhozi) [reply libs/app/profile/data-access/src/profile.state.ts:340]: Done

File: libs/app/forgot/ui/src/verification-modal/verification-modal.component.ts [hunk 7]
```diff
@@ -24,7 +25,7 @@ export class VerificationModalComponent {
 
     if (this.verification_code != "" && this.new_password != "" && this.confirm_password != "") {
       if (this.new_password != this.confirm_password) 
-        this.store.dispatch(new ShowError("Please Enter Matching Passwords"));
+        this.store.dispatch(new ShowInfo("Please Enter Matching Passwords"));
```
[the remark this answers is not in the document]
Comment (Shojiki-Lukhozi) [reply libs/app/forgot/ui/src/verification-modal/verification-modal.component.ts:28]: Done

File: libs/app/auth/data-access/src/auth.state.ts (hunk 1 above)
Comment (Shojiki-Lukhozi) [reply libs/app/auth/data-access/src/auth.state.ts:226]: Done

File: libs/app/auth/data-access/src/auth.state.ts (hunk 2 above)
Comment (Shojiki-Lukhozi) [reply libs/app/auth/data-access/src/auth.state.ts:180]: Done

File: libs/app/explore/data-access/src/explore.state.ts [hunk 8]
```diff
@@ -58,6 +59,8 @@ export class ExploreState {
                     explore: search,
                     recipes: data
                 });
+
+                this.store.dispatch(new ShowInfo('Results Found'));
```
[the remark this answers is not in the document]
Comment (Shojiki-Lukhozi) [reply libs/app/explore/data-access/src/explore.state.ts:63]: Done

File: libs/app/home/data-access/src/home.state.ts [hunk 9]
```diff
@@ -44,7 +45,7 @@ export class HomeState {
             });
         },
         error: error => {
-            this.store.dispatch(new ShowError('Failed to retrieved featured recipes'));
+            this.store.dispatch(new ShowInfo('Failed to retrieved featured recipes'));
```
[the remark this answers is not in the document]
Comment (Shojiki-Lukhozi) [reply libs/app/home/data-access/src/home.state.ts]: Done

**Pull request merged**